src/main/java/uk/gov/hmcts/reform/sscs/service/placeholders/FurtherEvidencePlaceholderService.java

```diff
@@ -37,7 +37,7 @@ public FurtherEvidencePlaceholderService(PlaceholderService placeholderService,
 
         Name name = getName(caseData, letterType);
         if (name != null) {
-            placeholders.put(NAME, name.getFullNameNoTitle());
+            placeholders.put(NAME, name.getFullNameNoTitle().substring(0, 40));
```

Comment:
I think it will handle 50 characters.
Would you need to truncated address lines as well?

Reply:
I'll run past Gareth. There is one spare address line now, so should be ok without truncating but I will confirm

Reply:
Updated the PR to truncate all address lines to 45 characters